src/emulator/host/src/host.cpp

```diff
@@ -75,7 +75,7 @@ bool init(HostState &state) {
 
     state.base_path = base_path.get();
     state.pref_path = pref_path.get();
-    state.window = WindowPtr(SDL_CreateWindow(window_title, SDL_WINDOWPOS_CENTERED, SDL_WINDOWPOS_CENTERED, 960, 544, SDL_WINDOW_OPENGL), SDL_DestroyWindow);
+    state.window = WindowPtr(SDL_CreateWindow(window_title, SDL_WINDOWPOS_CENTERED, SDL_WINDOWPOS_CENTERED, 960, 575, SDL_WINDOW_OPENGL), SDL_DestroyWindow);
```

Comment:
do 544 + whatever and make them both constants like so
`constexpr auto WINDOW_Y = 544;`
and
`constexpr auto WINDOW_Y_BORDER = whatever;`
and also use them to initialize host state (put them where ever you want, `GuiState` maybe?
so that you can use them in `sceGxmCreateContext` as I say below